request.lisp

```diff
@@ -127,7 +127,10 @@ body using the boundary BOUNDARY."
       (crlf))))
 
 (defun %read-body (stream element-type)
-  (declare (stream stream))
+  ;; On ABCL, a flexi-stream is not a normal stream. This is caused by
+  ;; a bug in ABCL which is supposedly quite difficult to fix. More
+  ;; details here: http://abcl.org/trac/ticket/377
+  #-abcl(declare (stream stream))
```

Comment:
I know it is a bit picky, but can you please put the `#+abcl` on a separate line?